Solutions/Prancer Log Integration/Analytic Rules/Vaults_High_Severity.yaml

```diff
@@ -0,0 +1,55 @@
+id: 0b76eef3-5dc0-41b1-9f67-fffa7783f5f6
+name: Vaults Alerts for Prancer
+description: |
+  'High severity vault alerts found by Prancer'
+severity: High
+requiredDataConnectors:
+- connectorId: PrancerLogData
+  dataTypes:
+    - prancer_CL
+queryFrequency: 1h
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 0
+eventGroupingSettings:
+  aggregationKind: SingleAlert
+status: Available
+tactics:
+  - Reconnaissance
+relevantTechniques:
+  - T1595
+query: |
+  union prancer_CL
+  | where deviceProduct_s == 'azure'V
```

Comment:
Hello @Phenox11, validation check is failing because of 'azure'V 
I think its a typo error could you please check

Reply:
Okay, I will double check.